frontend/src/scenes/sceneLogic.ts

```diff
@@ -236,20 +241,37 @@ export const sceneLogic = kea<sceneLogicType>({
         ],
     },
     urlToAction: ({ actions }) => {
+        const unmountFeatFlagLogic = featureFlagLogic.mount()
```

Comment:
Hmmm so that wasn't my experience – without mounting the logic here I was getting a "Can not find path "lib.logic.featureFlagLogic" in the store." error